packages/docker-engine/0002-oci-inject-kmod-in-all-containers.patch

```diff
@@ -0,0 +1,31 @@
+From e35f5eeeaa4c7b9ec1ae0720fc7de0fc4d43e02f Mon Sep 17 00:00:00 2001
+From: Arnaldo Garcia Rincon <agarrcia@amazon.com>
+Date: Thu, 30 May 2024 14:38:33 +0000
+Subject: [PATCH] oci: inject kmod in all containers
+
+Append a new mount to the default spec created for Linux containers
+
+Signed-off-by: Arnaldo Garcia Rincon <agarrcia@amazon.com>
```

Comment:
I sign off :+1:  Since this is slightly different than the patch that I wrote, you should add your `Signed-off-by` footer too.